Design/Rule0052and0053InternalProceduresNotReferenced.cs

```diff
@@ -6,47 +6,31 @@
 using Microsoft.Dynamics.Nav.CodeAnalysis.Symbols;
 using Microsoft.Dynamics.Nav.CodeAnalysis.Syntax;
 using Microsoft.Dynamics.Nav.CodeAnalysis.Packaging;
-
-
-
 using System.Collections.Immutable;
 
 namespace BusinessCentral.LinterCop.Design
 {
     [DiagnosticAnalyzer]
     public class Rule0052InternalProceduresNotReferencedAnalyzer : DiagnosticAnalyzer
     {
-
         private class MethodSymbolAnalyzer : IDisposable
         {
             private readonly PooledDictionary<IMethodSymbol, string> methodSymbols = PooledDictionary<IMethodSymbol, string>.GetInstance();
-
             private readonly PooledDictionary<IMethodSymbol, string> internalMethodsUnused = PooledDictionary<IMethodSymbol, string>.GetInstance();
             private readonly PooledDictionary<IMethodSymbol, string> internalMethodsUsedInCurrentObject = PooledDictionary<IMethodSymbol, string>.GetInstance();
             private readonly PooledDictionary<IMethodSymbol, string> internalMethodsUsedInOtherObjects = PooledDictionary<IMethodSymbol, string>.GetInstance();
-
             private readonly AttributeKind[] attributeKindsOfMethodsToSkip = new AttributeKind[] { AttributeKind.ConfirmHandler, AttributeKind.FilterPageHandler, AttributeKind.HyperlinkHandler, AttributeKind.MessageHandler, AttributeKind.ModalPageHandler, AttributeKind.PageHandler, AttributeKind.RecallNotificationHandler, AttributeKind.ReportHandler, AttributeKind.RequestPageHandler, AttributeKind.SendNotificationHandler, AttributeKind.SessionSettingsHandler, AttributeKind.StrMenuHandler, AttributeKind.Test };
-
             public MethodSymbolAnalyzer(CompilationAnalysisContext compilationAnalysisContext)
             {
                 NavAppManifest manifest = ManifestHelper.GetManifest(compilationAnalysisContext.Compilation);
 
                 if (manifest.InternalsVisibleTo != null && manifest.InternalsVisibleTo.Any())
-                {
```

Comment:
I am surprised to see the curly brackets getting removed though (and all the non-redundant whitespace) 😮 
It is good practice to have curly brackets in C# even if there is only a single statement.

Reply:
As a novice level C# developer I though I was improving readability of the code by removing the curly brackets. 

I thought if it is a one liner, the code is cleaner to have it without curly braces. Even maybe put it in one nice and short line. After all, C# compiler allows it, who am I to disagree with that? Boy, was I wrong 😯

https://www.armannotes.com/2020/11/23/why-you-should-always-use-curly-braces-for-single-statement-blocks-in-csharp/

@rvanbekkum, I genuinely appreciate you feedback on this. I've learned why you should always use curly braces for single statement blocks in C#. I'll revert these changes with a new PR.

Reply:
I suspect it wouldn't be as much of a problem with C# if it would always apply a formatter automatically like what happens for AL code. 🙂